controllers/clusteroperator.go

```diff
@@ -169,3 +203,25 @@ func (r *ProvisioningReconciler) updateCOStatusDegraded(degradedReason string, d
 
 	return r.syncStatus(co, conds)
 }
+
+// updateCOStatusAvailable updates the ClusterOperator's status to Available
+func (r *ProvisioningReconciler) updateCOStatusAvailable() error {
+	co, err := r.getOrCreateClusterOperator()
+	if err != nil {
+		return err
+	}
+
+	// Write the operand versions when available
+	co.Status.Versions = getOperandVersions()
+
+	conds := []osconfigv1.ClusterOperatorStatusCondition{
+		setStatusCondition(osconfigv1.OperatorAvailable, osconfigv1.ConditionTrue, string(ReasonEmpty),
+			fmt.Sprintf("Cluster Baremetal Operator is available at %s", r.printOperandVersions())),
+		setStatusCondition(osconfigv1.OperatorProgressing, osconfigv1.ConditionFalse, "", ""),
```

Comment:
I'm not sure using this function is really clearer than instantiating the resource inline here, but I don't feel so strongly that I'd block the PR over it.

Reply:
Perhaps a little bit more readable but I understand your point.